services/crypto_get_account_records.proto

```diff
@@ -43,6 +44,25 @@ message CryptoGetAccountRecordsQuery {
      * The account ID for which the records should be retrieved
      */
     AccountID accountID = 2;
+
+    /**
+     * If set, a restriction on the first payer record to be returned. The queried node will 
+     * return this record (assuming one qualifies), and up to to the next 179 records, in 
```

Comment:
Isn't the limit (180) configurable? In which case we shouldn't document 179 here, we should just say we will return all records up to some maximum amount (put it in better words, but that's the idea). Same with the 180 on line 54.

Reply:
Updated comment with link to the Services property.